app/assets/stylesheets/baskets.css.scss

```diff
@@ -41,3 +41,10 @@
     margin-bottom: 0;
   }
 }
+
+.empty {
+  background: $light-gray;
```

Comment:
I want to make sure that this will remove any background images or other background settings. I want the element to have a solid grey background.